React/src/index.tsx

```diff
@@ -13,7 +13,7 @@ root.render(
         url: "https://app.unleash-hosted.com/demo/api/frontend", // Your front-end API URL or the Unleash proxy's URL (https://<proxy-url>/proxy)
         clientKey:
           "demo-app:dev.95ae66ab673bf467facb68b2487904f4891064d26b47e89ca498063d", // A client-side API token OR one of your proxy's designated client keys
-        refreshInterval: 15, // How often (in seconds) the client should poll the proxy for updates
+        refreshInterval: 5, // How often (in seconds) the client should poll the proxy for updates
```

Comment:
```suggestion
        refreshInterval: 15, // How often (in seconds) the client should poll the proxy for updates
```
We don't want to encourage people to exceed default pooling interval. Things like this tend to stick until production and cause issues. I understand it's better for development to get faster feedback, but until we have dynamic back-off configurations we need to keep this value reasonable